test/Microsoft.Azure.ServiceBus.MessageId.Test/When_MessageIdPlugin_is_used.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
I copied this from the files already found in the solution.
Perhaps a separate PR should be issued to address the license change throughout the entire solution once this PR is merged? If not, at least provide the license text I should use. Thanks.